templates/sssd.conf.j2

```diff
@@ -19,7 +19,14 @@ ldap_id_mapping = True
 ignore_group_members = True
 fallback_homedir = {{ adjoin_sssd_homedir }}
 default_shell = {{ adjoin_sssd_default_shell }}
```

Comment:
To keep this file as clean as possible, can you put this in an if/else construction?

{% if domain.dyndns_update | default(false) %}
# new stuff
{% else %}
dyndns_update = false
{% endif %}